src/reactor.rs

```diff
@@ -583,7 +531,7 @@ impl<H: Handler, P: Poll> Runtime<H, P> {
                 #[cfg(feature = "log")]
                 log::trace!(target: "reactor", "Awoken by the controller");
 
-                reset_fd(&self.waker).expect("waker failure");
+                self.waker.reset();
```

Comment:
Is there anything else we can do than panic here? Maybe just log an error and keep going?

Reply:
Same question with the `waker failure` above

Reply:
My assumption that if the waker is broken, we enter UB territory: commands will not be relayed to the reactor from a service, and things will go unpredictable direction. Better to kill wounded horse here.